lib/govuk_navigation_helpers/content_item.rb

```diff
@@ -90,5 +99,32 @@ def hash
     def eql?(other)
       self == other
     end
+
+  private
+
+    def taxon_links
+      # A normal content item's taxon links are stored in ["links"]["taxons"]
+      # whereas a Taxon content item's taxon links are stored in ["links"]["parent_taxons"]
+      # so here we cater for both possibilities
+      Array(
+        content_store_response.dig("links", "taxons") ||
+          content_store_response.dig("links", "parent_taxons")
+      )
```

Comment:
I don't get what Array is doing here? If changing `nil` -> `[]` is what you are doing, would adding `|| []` to the end be clearer?